optaplanner-operator/pom.xml

```diff
@@ -160,6 +179,23 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>image</id>
```

Comment:
The image build does not work on Windows GHA, thus is cannot be a part of the `full` profile.

Reply:
Maybe add that comment directly to the POM?